src/Compilers/Core/CodeAnalysisTest/Symbols/DocumentationCommentIdTests.cs

```diff
@@ -154,6 +156,35 @@ End Namespace
             Assert.Null(DocumentationCommentId.GetFirstSymbolForDeclarationId("M:N.C.M``1(``1[])", comp));
         }
 
+        [Theory, CombinatorialData, WorkItem("https://github.com/dotnet/roslyn/issues/70159")]
+        public async Task TestReturnValueOnInvalidSymbol1(
+            [CombinatorialValues("int*", "dynamic")] string type)
+        {
+            var text = $$"""
+                class C
+                {
+                    unsafe void M({{type}} i)
+                    {
+                        var x = i + 1;
+                    }
+                }
+                """;
+
+            var comp = CreateCSharpCompilation(text, compilationOptions: TestOptions.UnsafeDebugDll).VerifyDiagnostics();
```

Comment:
>CreateCSharpCompilation [](http://example.com/codeflow?start=23&length=23)

Can we add a "similar" test for VB?

Reply:
Sure.  though i'm not knowledgable enough about the VB symbol model to know what would work here.  I used dynamic and pointers in c# because i knew about problems there.  But i don't think VB has either of those.  Is there another type of non-named type symbol that vb has where operators are synthesized for it?